ee-9/common/src/main/resources/license/preview-feature-pack-common-licenses.xml

```diff
@@ -809,7 +809,7 @@
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>
-      <artifactId>wildfly-weld-ejb-jakarta</artifactId>
+      <artifactId>wildfly-weld-ejb</artifactId>
```

Comment:
This is ok, but I filed https://issues.redhat.com/browse/WFLY-17121 to remove the need for this. I'm not asking for a change in this PR though because there are many others that need change as well.